easybuild/easyconfigs/b/BBMap/BBMap-38.26-foss-2018b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '38.26'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['e439709d6d294aae5938cb253c8640f8e0dfeb227d5b01ce17c37e9871a458b4']
+
+dependencies = [('Java', '1.8.0_181', '', True)]
```

Comment:
@fizwit We should change this to use the `Java/1.8` wrapper:

```python
dependencies = [('Java', '1.8', '', True)]
```

(I was assuming the tests would fail if that's not the case, I need to check why that's not the case...)